common/src/main/resources/emails/SiteIssueReport.html

```diff
@@ -0,0 +1,394 @@
+                                                    </tr>
+                                                    </tbody>
+                                                </table>
+                                            </td>
+                                        </tr>
+                                        </tbody>
+                                    </table>
+
+                                    <!-- Title -->
+                                    <!--[if mso]>
+                                    <table style="width: 100%; border-collapse: collapse; border: 0">
+                                        <tr>
+                                            <td style="padding: 10px 40px; font-family: Tahoma, sans-serif">
+                                    <![endif]-->
+                                    <div style="color:#555555;font-family:Montserrat, Trebuchet MS, Lucida Grande, Lucida Sans Unicode, Lucida Sans, Tahoma, sans-serif;line-height:1.2; padding: 10px 40px">
+                                        <div style="line-height: 1.2; font-size: 12px; color: #555555; font-family: Montserrat, Trebuchet MS, Lucida Grande, Lucida Sans Unicode, Lucida Sans, Tahoma, sans-serif; mso-line-height-alt: 14px;">
+                                            <p style="font-size: 24px; line-height: 1.2; text-align: center; word-break: break-word; mso-line-height-alt: 36px; margin: 0;"><span style="font-size: 30px; color: #2b303a;"><strong>A user found an issue with a site</strong></span></p>
+                                        </div>
+                                    </div>
+
+                                    <!-- Paragraph -->
+                                    <!--[if mso]>
+                                    </td>
+                                    </tr>
+                                    </table>
+                                    <![endif]-->
+                                    <!--[if mso]>
+                                    <table style="width: 100%; border-collapse: collapse; border: 0">
+                                        <tr>
+                                            <td style="padding: 10px 30px; font-family: Tahoma, sans-serif">
+                                    <![endif]-->
+                                    <div style="color:#555555;font-family:Montserrat, Trebuchet MS, Lucida Grande, Lucida Sans Unicode, Lucida Sans, Tahoma, sans-serif;line-height:1.5; padding: 10px 30px">
+                                        <div style="line-height: 1.5; font-size: 15px; text-align: center; font-family: Montserrat, Trebuchet MS, Lucida Grande, Lucida Sans Unicode, Lucida Sans, Tahoma, sans-serif; color: #808389; mso-line-height-alt: 18px;">
+                                                ${name} (${email}) found the following issue with <a href="${siteLink}" target="_blank">site ${siteId}</a>: <b>${reportReason}</b>.
```

Comment:
Just curious-- how did you get the siteLink variable here? I don't see it being passed with the templateValues hash map but in your screenshot I assume that it works and links to that site's page?

Reply:
Good catch! I'm actually not passing `siteLink` in at all, so the link is broken. I didn't actually test the link and just assumed it worked, so that's my bad here.